functions/annotation_request_handler.scm

```diff
@@ -22,7 +22,7 @@ val-msg ))
 (define (gene-info genes)
     (let ([info '()])
          (for-each (lambda (g)
-              (set! info (append info (list (node-info g))))
+              (set! info (append info (list (ListLink (node-info g)))))
```

Comment:
You dont need to make the node-info results in ListLink here, there were supposed to be returned as ListLink in the node-info function. It creates inconsistency for others